netty/src/test/java/io/grpc/netty/NettyChannelBuilderTest.java

```diff
@@ -90,6 +93,24 @@ public void failInvalidAuthority() {
   }
 
   @Test
+  public void sslContextCanBeNull() {
+    NettyChannelBuilder builder = mock(NettyChannelBuilder.class);
+    builder.sslContext(null);
```

Comment:
Hmm, that's tricky though.   Using the fact that Mockito won't override a final method is pretty niche, and not obvious from reading just the test body.   Using a real NCB would make your intent clear.